subprojects/workers/src/main/java/org/gradle/workers/internal/WorkerDaemonServer.java

```diff
@@ -113,7 +113,7 @@ public String toString() {
 
     private static class WorkerDaemonServices extends WorkerSharedUserHomeScopeServices {
 
-        // TODO: configuration-cache - deprecate workers access to ProviderFactory?
+        // TODO:configuration-cache - deprecate workers access to ProviderFactory?
```

Comment:
I feel like this TODO is either not necessary or missing a real issue reference. Otherwise it has a high chance of becoming rot ...

Reply:
It's ok to leave it as we screen the codebase for those every now and then.